staging/src/k8s.io/apimachinery/pkg/runtime/schema/group_version.go

```diff
@@ -136,7 +136,11 @@ func (gvk GroupVersionKind) GroupVersion() GroupVersion {
 }
 
 func (gvk GroupVersionKind) String() string {
-	return gvk.Group + "/" + gvk.Version + ", Kind=" + gvk.Kind
+	if len(gvk.Group) == 0 {
```

Comment:
revert this change, being able to tell the difference between version and group in the output is useful